.github/workflows/notify-about-breaking-changes.yml

```diff
@@ -0,0 +1,71 @@
+name: "Notify about breaking changes"
+
+on:
+  push:
+    branches:
+      - v1.x
+    paths:
+      - ".changelog/unreleased/breaking-changes/**"
+
+jobs:
+  notify:
+    name: Notify about breaking changes
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: 'v1.x'
+
+      - name: Detect new files in directory
+        id: detect-new-files
+        run: |
+          DIRECTORY=".changelog/unreleased/breaking-changes"
+
+          # Get a list of new files in the directory (added in the last commit)
+          NEW_FILES=$(git diff --name-status HEAD^..HEAD $DIRECTORY | grep "^A" | awk '{print $2}')
+
+          if [ -n "$NEW_FILES" ]; then
+            echo "New files detected: $NEW_FILES"
+            echo "::set-output name=new_files::$NEW_FILES"
+          else
+            echo "No new files detected"
+            echo "::set-output name=new_files::none"
+          fi
+
+      - name: Read the content of new files
+        if: steps.detect-new-files.outputs.new_files != 'none'
+        id: read-content
+        run: |
+          NEW_FILES="${{ steps.detect-new-files.outputs.new_files }}"
+          CONTENT=""
+
+          for FILE in $NEW_FILES; do
+            FILE_CONTENT=$(cat $FILE)
+            CONTENT="$CONTENT\n---\n$FILE_CONTENT\n"
+          done
+
+          echo "$CONTENT"
+          echo "::set-output name=file_content::$CONTENT"
+
+      - name: Post to a Slack channel
+        id: slack
+        if: steps.read-content.outputs.file_content != ''
+        uses: slackapi/slack-github-action@v1.27.0
+        with:
+          channel-id: 'C03Q5J9SXS8' # cometbft-engineering
+          # channel-id: 'shared-sdk-comet'
```

Comment:
I am assuming that the sdk channel is commented out to test the engineering channel first and if it works we add to the sdk channel so all good